.env.example

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Added an example `.env` file (that gets copied over for `make`/`make oss`) for easier OSS configuration.